etc/scripts/sync-configured-state-from-csv.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+
+"""
+Script to set isConfigured value against Jira App properties from CSV of JiraHosts.
+
+Takes an installationIds.csv with installationIds.
```

Comment:
Just a question here: Does this mean that we need another extra step of getting the list of existing installation's id in a CSV?

Reply:
yup, I have a list already, so we can do that.

Alternatively, we could say run for N then stop  and maintain either and index of where we ended or maintain the output.csv as current and it will skip syncs installs.

Just needed a easy way to stop and start incase of issues basically.